docs/variants/asrock_spc741d8/recovery.md

```diff
@@ -0,0 +1,79 @@
+# Recovery
+
+## Intro
+
+The following documentation describes the process of recovering hardware from
+the brick state using an [RTE](../../transparent-validation/rte/introduction.md)
+and Dasharo open-source firmware.
+
+## External flashing
+
+### Prerequisites
+
+* [Prepared RTE](../../transparent-validation/rte/v1.1.0/quick-start-guide.md)
+* 5x2 1mm pitch female header to 10x 2.54mm female DuPont connector adapter
+  cable
+* TPM removed
+
+### Connections
+
+To prepare the stand for flashing follow the steps described in
+the [Generic test stand setup](../../unified-test-documentation/generic-testing-stand-setup.md#detailed-description-of-the-process)
```

Comment:
nit (missing dot):
```suggestion
the [Generic test stand setup](../../unified-test-documentation/generic-testing-stand-setup.md#detailed-description-of-the-process).
```
It just stands out because sentences without links do have dots.  And we're probably copying such typos.